script.js

```diff
@@ -1,23 +1,42 @@
 var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
 var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
+var cardValues = [];
 var game = {
   deck: [],
   players: [],
   hands: [],
-  buildDeck: function(){
 
+  buildDeck: function(){
+    for (i = 0; i < values.length; i++){
+      game.deck.push([values[i], "Clubs"]);
+      game.deck.push([values[i], "Hearts"]);
+      game.deck.push([values[i], "Diamonds"]);
+      game.deck.push([values[i], "Spades"]);
+    }
   },
+//Stolen from the internet. Using this so I can move on. Will revisit if I have time (I'm not gonna have time tonight 😞).//
   shuffleDeck: function(){
-
+    	for(var j, x, i = game.deck.length; i; j = parseInt(Math.random() * i), x = game.deck[--i], game.deck[i] = game.deck[j], game.deck[j] = x);
```

Comment:
Dang. That is one gnarly `for` loop. I'm impressed -- but would perhaps recommend not writing it this way since it's hard to ready!